tests/unit/cli/registry/test_cli_registry.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+import pytest
+
+from otx.cli.registry import Registry, find_and_parse_model_template
+from tests.test_suite.e2e_test_system import e2e_pytest_unit
+
+
+class TestRegistry:
+    @pytest.fixture
+    def mock_templates(self, mocker):
+        template1 = mocker.MagicMock(framework="OTX001", task_type="Classification", model_template_id="001")
+        template2 = mocker.MagicMock(framework="OTX002", task_type="Segmentation", model_template_id="002")
+        template3 = mocker.MagicMock(framework="OTX001", task_type="Detection", model_template_id="003")
+        return [template1, template2, template3]
+
+    @e2e_pytest_unit
+    def test_init(self, mocker, mock_templates):
+        mock_glob = mocker.patch("glob.glob")
+        mock_glob.return_value = ["./template.yaml"]
+        mock_abspath = mocker.patch("os.path.abspath")
+        mock_abspath.return_value = "./template.yaml"
+        # mock_exists = mocker.patch("os.path.exists")
+        mock_parse_model_template = mocker.patch("otx.cli.registry.registry.parse_model_template")
+        mock_parse_model_template.return_value = mock_templates[0]
+
+        with pytest.raises(RuntimeError):
```

Comment:
I think it's better to separate this part as other test function because this is different with working well case. How do you think about?

Reply:
I wonder if it is necessary to use duplicate functions with the same mocking as above. What do you think?

Reply:
Yes. Because I think one test case should test a single test. To remove duplication, you can use fixture as below, I think.
```

@pytest.fixture
def setup_init_test(self, mocker):
        mock_glob = mocker.patch("glob.glob")
        mock_glob.return_value = ["./template.yaml"]
        mock_abspath = mocker.patch("os.path.abspath")
        mock_abspath.return_value = "./template.yaml"
        # mock_exists = mocker.patch("os.path.exists")
        mock_parse_model_template = mocker.patch("otx.cli.registry.registry.parse_model_template")
        mock_parse_model_template.return_value = mock_templates[0]
```

Reply:
This has been reflected.